src/main/kotlin/blackjack/domains/participants/Dealer.kt

```diff
@@ -0,0 +1,50 @@
+package blackjack.domains.participants
+
+import blackjack.GameRule.Companion.BLACKJACK
+import blackjack.GameRule.Companion.DEALER_DRAW_CONDITION
+import blackjack.GameScoreType
+import blackjack.domains.deck.Cards
+import blackjack.views.Output.printDealerDraw
+
+class Dealer(
+    override val name: String = "딜러",
+    override val cards: Cards = Cards()
+) : User(name, cards) {
+
+    private val gameScores = mutableListOf<GameScoreType>()
+
+    override fun isDrawable(): Boolean {
+        if (!cards.isDrawable(DEALER_DRAW_CONDITION)) return false
+        return true
+    }
+
+    override fun printHasCards() {
+        printDealerDraw()
+    }
+
+    override fun win() {
+        gameScores.add(GameScoreType.WIN)
+    }
+
+    override fun lose() {
+        gameScores.add(GameScoreType.LOSE)
+    }
+
+    override fun draw() {
+        gameScores.add(GameScoreType.DRAW)
+    }
+
+    fun isOver21(): Boolean {
```

Comment:
```suggestion
    fun isOverBlackjack(): Boolean {
```
21이라는 단어보다는 Blackjack 이라는 단어를 사용하는 것이 더 가독성이 좋아요.

Reply:
감사합니다~